data/transition-sites/directgov_volunteering.yml

```diff
@@ -0,0 +1,9 @@
+---
+site: directgov_volunteering
+whitehall_slug: government-digital-service
+homepage: https://www.gov.uk/government/get-involved/take-part/volunteer
+tna_timestamp: 20130430103829
+host: volunteering.direct.gov.uk
+css: directgov
+global: =301 https://www.gov.uk/government/get-involved/take-part/volunteer
+title: 'Volunteering'
```

Comment:
`css` and `title` won't ever be used if the site is a global redirect, but there's no harm in keeping them, and if we change it in future to not be a global redirect then they'd be needed.